src/configs/index.js

```diff
@@ -11,6 +11,11 @@ export const CART_AND_WISHLIST_IMAGES = {
   darkTheme: './images/cart/empty-cart-black-theme-image.png'
 };
 
+export const EMPTY_PRODUCT_LIST_IMAGES = {
+  lightTheme: '../images/cart/empty-cart-light-theme-image.png',
```

Comment:
If only the name of the file is changing better to move file name to constants, also the image shouldn't be the SVG file?

Reply:
Fixed this.